tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -2879,6 +2879,11 @@ public function createPriceSetWithPage($entity = NULL, $params = []): void {
       'label' => 'Goat Breed',
       'html_type' => 'Radio',
     ]);
+    $addOnPriceField = $this->callAPISuccess('price_field', 'create', [
+      'price_set_id' => $priceSetID,
+      'label' => 'Goat Addons',
```

Comment:
everyone needs to accessorize their goat

Reply:
![Uploading image.png…]()

Reply:
Greatest of All Tiaras